module/os/freebsd/zfs/arc_os.c

```diff
@@ -204,8 +204,7 @@ arc_memory_throttle(spa_t *spa, uint64_t reserve, uint64_t txg)
 uint64_t
 arc_free_memory(void)
 {
-	/* XXX */
-	return (0);
+	return (PAGESIZE * freemem);
```

Comment:
I'd add type conversion to uint64_t here, since freemem is a define for vm_free_count(), which return u_int.  Or use ptob().

Reply:
ptob is `#define     ptob(x)         ((x) * PAGESIZE)` so I guess I'll go with both 👍